uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/braintree/BraintreePaymentPresenter.kt

```diff
@@ -107,7 +107,7 @@ class BraintreePaymentPresenter(view: PaymentMVP.View,
                 passBackNonce(braintreeSDKToken)
             }
         } else {
-            view?.showPaymentUI(braintreeSDKToken)
+            view?.showPaymentUI(braintreeSDKToken, null, null)
```

Comment:
I guess you don't need to explicitly pass null.

Reply:
I've set as default of null and removed this.